### ISSUE TITLE
OAuth is not defined

### ISSUE THREAD

**Issue by Dandrass** [icbestCA/giftmanager#28]
Hello there, I'm liking your project a lot, used it for a couple of weeks, but i'm having difficulties booting it up via docker compose after the latest upgrade.

I got this error : 

`NameError: name 'OAuth' is not defined
giftmanager  | Traceback (most recent call last):
giftmanager  |   File "/app/app.py", line 139, in <module>
giftmanager  |     oauth = OAuth(app)`

I think it's related to #4 but i have not setup an OIDC solution yet.
Is it possible to still access the app without OIDC or is it mandatory now ?

**Comment by vfedetz** [comment]
I am also getting the same error. The docker container was up and working a few days ago. Now it fails to properly start with error:

```
NameError: name 'OAuth' is not defined

Traceback (most recent call last):

  File "/app/app.py", line 139, in <module>

    oauth = OAuth(app)

            ^^^^^
```

**Comment by vfedetz** [comment]
There does not seem to be any releases or docker tags available for this project besides "latest". I am not sure how to roll back to a previous version of the docker image.

**Comment by icbestCA** [comment]
Yeah, sorry, I forgot to import the authlib in the app.py . I will correct this in the next 8 hours. I also will find a way to add versioning to the app so it is easier to go back if needed.
Thanks for the feedback.
Sorry again.

**Comment by icbestCA** [comment]
See fixes in the release 1.0.1
Will now work properly with OIDC.
Added versioning to the GitHub and Docker. The latest tag is also up to date.
